src/components/pages/AdminHome/AdminCourseCard.js

```diff
@@ -20,19 +19,28 @@ export default function CourseCard(props) {
     <div className="admin-course-card-container">
       <Card title={course.course_name}>
         <h3>
-          {headers[17].toUpperCase()}{' '}
-          <span className="black">{course.program_name}</span>
+          Program:
+          <span className="black">{' ' + course.program_name}</span>
```

Comment:
Is there a reason that a space is needed here? If this is being used to position text CSS would be a better, more clear option.